scripts/jenkins/qa_openstack.sh

```diff
@@ -446,18 +446,45 @@ if [ -e /etc/tempest/tempest.conf ]; then
     fi
 
     pushd /var/lib/openstack-tempest-test/
+
+    # Handle OpenStack release specific blacklisting of known to fail tests
+    blacklist_args=""
+    case "${cloudsource}" in
+    (openstackrocky)
```

Comment:
It is however perfectly acceptable and valid bash syntax, as per bash documentation:

       case word in [ [(] pattern [ | pattern ] ... ) list ;; ] ... esac
              A case command first expands word, and tries to match it against each pattern in turn, using the same matching rules as for pathname expansion (see Pathname Expansion below).  The word is expanded using tilde expansion, parameter and variable expansion, arith‐
              metic expansion, command substitution, process substitution and quote removal.  Each pattern examined is expanded using tilde expansion, parameter and variable expansion, arithmetic expansion, command substitution, and process substitution.  If the nocasematch
              shell option is enabled, the match is performed without regard to the case of alphabetic characters.  When a match is found, the corresponding list is executed.  If the ;; operator is used, no subsequent matches are attempted after  the  first  pattern  match.
              Using  ;&  in  place  of  ;; causes execution to continue with the list associated with the next set of patterns.  Using ;;& in place of ;; causes the shell to test the next pattern list in the statement, if any, and execute any associated list on a successful
              match.  The exit status is zero if no pattern matches.  Otherwise, it is the exit status of the last command executed in list.